crates/solver/src/driver.rs

```diff
@@ -170,7 +170,13 @@ impl Driver {
                         Ok(inner) => inner.map_err(SolverRunError::Solving),
                         Err(_timeout) => Err(SolverRunError::Timeout),
                     };
-                metrics.settlement_computed(solver.name(), start_time);
+                let response = match &result {
+                    Err(SolverRunError::Timeout) => "solution",
```

Comment:
Shouldn't this be `none` (or a dedicated `timeout`)? The description for `solution` is:
> solver worked on optimizing a non trivial auction

Which doesn't exactly fit IMO (especially since the solver is producing an empty solution).

Also, in general, splitting it into a separate `timeout` label lets us do Prometheus queries where `response=~"solution|timeout"` to include both `solution` and `timeout` categories in specific cases (but also leaves us the option to consider them separately if it makes more sense in other contexts).

Reply:
I definitely want to group timeout and solution separately from none but I think you are right that the grouping should probably rather happen in grafana to be more "correct" in the code.